public/index.html

```diff
@@ -13,6 +13,16 @@
     <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
     <meta name="msapplication-TileColor" content="#2d89ef">
     <meta name="theme-color" content="#ffffff">
+    <meta property="og:title" content="MonoMotion Robotics Framework" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://monomotion.netlify.com" />
+    <meta property="og:image" content="https://monomotion.netlify.com/static/media/trainer.5ff8e856.png" />
```

Comment:
this static image is generated by webpack and there is no guarantee that this image always exists.
We should add image directly in `public/` and refer that image in ogp and twitter card

Reply:
Sorry, I'll fix.